tools/buildsys/src/manifest.rs

```diff
@@ -504,6 +504,7 @@ fn is_manifest_type(pkg_metadata: &PackageMetadata, manifest_type: BuildType) ->
         BuildType::Package => metadata_table.get("build-package").is_some(),
         BuildType::Kit => metadata_table.get("build-kit").is_some(),
         BuildType::Variant => metadata_table.get("build-variant").is_some(),
+        BuildType::Repack => metadata_table.get("build-variant").is_some(),
```

Comment:
This would be better as an error or an `unreachable!` - we shouldn't be taking this graph-walk path for a repack operation, and the longer we make it work before failing, the harder it will be to diagnose the logic error that let us reach this point.

Reply:
Since this function returns a bool, `unreachable!` might be cleanest.